db/db.login/db.login.html

```diff
@@ -15,8 +15,8 @@ <h2>NOTE</h2>
 file in the user account, specifically
 
 <ul>
-<li> in the 'home' directory, i.e. <tt>$HOME/.grass7/dblogin</tt> (Unix-like systems)</li>
-<li> <tt>%APPDATA%/GRASS7/dblogin</tt> (MS-Windows)</li>
+<li> in the 'home' directory, i.e. <tt>$HOME/.grass8/dblogin</tt> (Unix-like systems)</li>
+<li> <tt>%APPDATA%\GRASS8\dblogin</tt> (MS-Windows)</li>
```

Comment:
wouldn't it be

`C:\Users\youruser\AppData\Roaming\GRASS7`

?

Reply:
in this case `%APPDATA%\Roaming\GRASS8`

Reply:
ok, fixed in 9ee56ed3c